### ISSUE TITLE
Operator functions not working when lodata is configured for readonly

### ISSUE THREAD

**Issue by ianket** [flat3/lodata#196]
For a project I'm using the RepositoryInterface from lodata to use as ground for my Operator functions. This project I am using this on is used as a read-only project, but has some operator functions to pass some data to an external source. But when trying to create this I noticed that operator functions always throw the "This service is read-only" message when executed (when readonly=true in the config, works when readonly=false). 

Now I was wondering if this was indeed intended this way or not, because I'd prefer to only expose a few operator endpoints instead of the entire Create/Update/Delete function stack.

Thanks in advance for the help!

**Comment by 27pchrisl** [comment]
Hi @ianket, you're right there was not a way to do it as you've described. As the code was written you can disallow an activity in read/write mode, but you couldn't allow an activity in read-only mode.

I have resolved this in v5.3.1, so you can enable the `authorization` feature to turn on gates in `config/lodata.php`, and then whitelist the operation by its name in your application:

```php
\Illuminate\Support\Facades\Gate::define('lodata', function (?User $user, \Flat3\Lodata\Helper\Gate $gate) {
  return 'myoperation' === $gate->getResource()->getName();
});
```

**Comment by ianket** [comment]
It does seem to work, but I noticed that when `'authorization' => false`, all my requests end up with `This request is not permitted`. But once I simply add (to make everything allowed):
```php
\Illuminate\Support\Facades\Gate::define('lodata', function (?User $user, \Flat3\Lodata\Helper\Gate $gate) {
    return true;
});
```
then it starts to work again (even though authorization => false). 

Once I set authorization to true and make the gate allow the operation, then I can indeed use the operation while in read-only mode so that does seem to work correctly.
